arbeitszeit_flask/templates/macros/plan_details.html

```diff
@@ -151,6 +176,9 @@ <h1 class="title">
                 </div>
                 <div class="tile is-parent">
                     <div class="tile is-child notification has-background-warning-light">
+                        <div class="has-text-right has-text-info-dark">
+                            {{ info_tooltip(id="id_labour_per_unit", text=gettext("This is to account for real working hours per unit, independent from consumption price which could be reduced, e.g. for public plans.")) }}
```

Comment:
1) typo: "is THE account"
2) Actually there are only two reasons why the "price" differs from the costs: Either the plan is part of what we call a "price cooperation" [Grundprinzipien 1930, p. 101: "Die horizontale Zusammenfassung ist zunächst nur eine rechnerische Angelegenheit zur Feststellung der gesellschaftlich durchschnittlichen Produktionszeit und im Zusammenhang damit des Grades der Produktivität der einzelnen Betriebe. Sicher kann und darf es dabei nicht bleiben, sondern es muß auch zu gegenseitiger technischer Durchdringung kommen."] OR the plan is a public plan, then it is zero.
As the info text is now, it appears as if there could be several, and arbitary, reasons why the "price could be reduced".

Reply:
done